python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py

```diff
@@ -263,12 +265,28 @@ def predict_dygraph(args, batch_generator):
                 eos_id=args.eos_idx,
                 beam_size=args.beam_size,
                 max_len=args.max_out_len)
-            finished_seq = finished_seq.numpy()
-            finished_scores = finished_scores.numpy()
+            seq_ids = seq_ids.numpy()
+            seq_scores = seq_scores.numpy()
+            if step_idx % args.print_step == 0:
+                if step_idx == 0:
+                    logging.info(
+                        "Dygraph Predict: step_idx: %d, 1st seq_id: %d, 1st seq_score: %.2f"
+                        % (step_idx, seq_ids[0][0][0], seq_scores[0][0]))
+                    avg_batch_time = time.time()
+                else:
+                    speed = args.print_step / (time.time() - avg_batch_time)
+                    speed_list.append(speed)
+                    logging.info(
+                        "Dygraph Predict: step_idx: %d, 1st seq_id: %d, 1st seq_score: %.2f, speed: %.3f step/s"
```

Comment:
Not import English knowledge: "step/s" -> "steps/s"

Reply:
Thanks. I will modify it next PR.